src/hydra/fetchHydra.js

```diff
@@ -30,8 +30,11 @@ export default (url, options = {}) => {
     const status = data.response.status;
 
     if (status < 200 || status >= 300) {
+      const body = data.body;
+      delete body.trace;
```

Comment:
shouldn't we log this though?

Reply:
It will be logged by backend + you'll be able to see it in Network Tab, but for Production Mode there won't be any trace